model/task_lifecycle.go

```diff
@@ -286,6 +293,21 @@ func getStepback(taskId string, project *Project) (bool, error) {
 
 // doStepBack performs a stepback on the task if there is a previous task and if not it returns nothing.
 func doStepback(t *task.Task) error {
+	if t.DisplayOnly {
+		catcher := grip.NewSimpleCatcher()
+		for _, et := range t.ExecutionTasks {
+			execTask, err := task.FindOne(task.ById(et))
```

Comment:
are there going to be cases where a single display task has multiple execution tasks?

Reply:
they'll almost always have multiple execution tasks. I switched it to not be multiple findOnes if that was your concern